app/lib/todos.server.ts

```diff
@@ -54,4 +55,18 @@ export function deleteTodo(id: string): boolean {
     return true;
   }
   return false;
+}
+
```

Comment:
issue (blocking): in-memory store is for demo only single session and all todos will be lost in server restart. We need database and define access roles- who can edit at what time. Redis can be used for broadcasting board event updates and will work across multiple servers and used before, during and after database transactions.
```suggestion
// TODO: Add new feature that allows editing one item or board at the time, e.g. temporary lock with redis or database transactions. It will prevent concurrency and race conditions
```